docs/validation.md

```diff
@@ -0,0 +1,102 @@
+# Validation
+
+Prism offers multiple validation layers that help you identify potential issues with your client ahead of the time.
+
+## Parameters validation
+
+Requests to operations which expect a request body, a query parameter or a path parameter will be validated.
+
+For example: let's make a a POST with a JSON body which is missing the required `name` parameter. We'll be using the [petstore openapi 2 version][petstore-oas2]
+
+```bash
+curl -X POST -s -D "/dev/stderr" -H "content-type: application/json" -d '{"tag":"Stowford"}' http://127.0.0.1:4010/pets | json_pp
+```
+
+In such case, Prism will:
+
+* Look for a response with status code `422` on the operation you were trying to execute
+* If there's not a `422` defined, it will look for a response with status code `400` on the operation you were trying to execute
+* In case there's nor a `422` nor a `400` defined, Prism will create a `422` response code for you indicating all the validation errors it found along the way. Such response will have a payload conforming the [application/problem+json][rfc7807] specification.
```

Comment:
```suggestion
* In case there's neither a `422` nor a `400` defined, Prism will create a `422` response code for you indicating all the validation errors it found along the way. Such response will have a payload conforming the [application/problem+json][rfc7807] specification.
```